app/controllers/api/v1/release_packages/pypi/simple_controller.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Api::V1::ReleasePackages
+  class Pypi::SimpleController < Api::V1::BaseController
+    include Rendering
+
+    before_action :scope_to_current_account!
+    before_action :require_active_subscription!
+    before_action :authenticate_with_token
+
+    def index
+      products = authorized_scope(apply_scopes(current_account.products.pypi)).limit(1_000)
+      packages = ReleasePackage.for(products)
+      authorize! packages,
+        with: ReleasePackagePolicy # can't be inferred when empty
+
+      render 'api/v1/release_packages/pypi/simple/index',
+        layout: 'layouts/plain',
+        locals: {
+          account: current_account,
+          packages:,
+        }
+    end
+
+    def show
+      product = FindByAliasService.call(
+        authorized_scope(current_account.products.pypi),
+        id: params[:id],
+        aliases: :code,
+      )
+
+      artifacts = authorized_scope(apply_scopes(product.release_artifacts.pypi)).limit(1_000)
```

Comment:
Is a limit of `1_000` high enough? Looking at projects like Rails and FastAPI, it's big enough, but projects like React have > 1,000 versions. We can always change this, of course. Just something to note.